src/python/WMComponent/DBS3Buffer/DBSUploadPoller.py

```diff
@@ -775,7 +778,7 @@ def retrieveBlocks(self):
         for result in blocksToClose:
             # Remove from list of work being processed
             self.queuedBlocks.remove(result.get('name'))
-            if result["success"] == "uploaded":
+            if result["success"] == "uploaded" or result["success" == "already exist"]:
```

Comment:
This line is clearly wrong. But in any case, I would stick with `uploaded` value only and keep things as simple as we can. In the end, uploaded means that the block has already been uploaded (same as already exists).

Reply:
ok, removed and use "uploaded" in both case.